exercises/rotational-cipher/cannonical-data.json

```diff
@@ -32,7 +32,7 @@
                 "expected": "z"
```

Comment:
I think @petertseng mentioned this but I see it still here.  Shouldn't `rotate a by 0 same input as output` be worded `rotate a by 0, same output as input` and then shouldn't the `shift_key` be equal to 0 instead of 26.  If left like it is wouldn't this test also be considered a wrap around test?